benchmark/launch_benchmark.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+# This script will preprocess files (determine orthologous genes, compute
+# their trees and MSAs, run BLASTP on the query genome) and passes this data
+# to launch_software.sh for executing HGT detection software. For fields
+# that are empty, None should be passed.
+
+# working dir
+working_dir=$(readlink -m $1)
+# scripts dir
+scripts_dir=$(readlink -m $2)
+# species tree in Newick format
+species_tree_fp=$3
+# species raw genome in FASTA format
+species_genome_fp=$4
+# species HMM model (produced by GeneMarkS)
+species_model_fp=$5
+# query species protein coding sequences in FASTA format
+query_species_coding_seqs_fp=$6
+# reference species protein coding sequences in FASTA format
+ref_species_coding_seqs_fp=$7
+# gene trees in Newick format
+gene_tree_dir=$8
+# gene multiple sequence alignment dir
+gene_msa_dir=$9
+# tabular DIAMOND alignments of query genome
+diamond_tabular_query_fp=${10}
```

Comment:
Shell interprets $10 as $1 unless there are brackets around ${10} [stackoverflow](http://stackoverflow.com/questions/4816635/how-do-i-access-arguments-to-functions-if-there-are-more-than-9-arguments)